packages/amplify-app/src/index.js

```diff
@@ -338,6 +339,36 @@ async function createAndroidHelperFiles() {
   }
 }
 
+async function createIosHelperFiles() {
+  const configFile = path.join(process.cwd(), '/amplifyxc.config');
+  const configStr = 'push=false\nmodelgen=false\nprofile=default\nenvName=amplify';
+  const awsConfigFile = path.join(process.cwd(), '/awsconfiguration.json');
+  const amplifyConfigFile = path.join(process.cwd(), '/amplifyconfiguration.json');
+  const amplifyDir = path.join(process.cwd(), '/amplify');
+  const configJsonObj = {};
+  const configJsonStr = JSON.stringify(configJsonObj);
+
+  // Write files if needed and them to xcode project if one exists
+  if (!fs.existsSync(configFile)) {
+    fs.writeFileSync(configFile, configStr);
+  }
+  await addFileToXcodeProj(configFile);
+
+  if (!fs.existsSync(awsConfigFile)) {
+    fs.writeFileSync(awsConfigFile, configJsonStr);
+  }
+  await addFileToXcodeProj(awsConfigFile);
+
+  if (!fs.existsSync(amplifyConfigFile)) {
+    fs.writeFileSync(amplifyConfigFile, configJsonStr);
+  }
+  await addFileToXcodeProj(amplifyConfigFile);
```

Comment:
Would adding the same file mutliple time using `addFileToXcodeProj` have any problems? Does it dedupe?

Reply:
The xcode helper dedupes. Also, the script reaches here only one time since it checks if amplfyxc.config exists.